eng/Packages.Data.props

```diff
@@ -186,7 +186,7 @@
     <PackageReference Update="ApprovalTests" Version="3.0.22" />
     <PackageReference Update="ApprovalUtilities" Version="3.0.22" />
     <PackageReference Update="Azure.Identity" Version="1.7.0" />
-    <PackageReference Update="Azure.Messaging.EventHubs" Version="5.2.0" />
+    <PackageReference Update="Azure.Messaging.EventHubs" Version="5.8.1" />
```

Comment:
Do we know why we need these versions to be different from the others listed above?

Reply:
We do not, and the existence of these overrides is something that we're discussing currently.   I would like to see them go away.